packages/lib-react-components/src/Media/components/Video/Video.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Video as GrommetVideo } from 'grommet'
+
+export default function Video({ alt, controls, fit, src }) {
+  const controlsOption = (controls) ? 'below' : false
+  return (
+    <Box>
+      <GrommetVideo a11yTitle={alt} controls={controlsOption} fit={fit} preload='metadata' src={src} />
```

Comment:
Probably not a big deal, but should we use the same props for audio and video, so add `preload` to the Audio component?